x-pack/plugins/alerting/public/hooks/use_breadcrumbs.ts

```diff
@@ -72,19 +72,15 @@ export const useBreadcrumbs = (pageDeepLink: MaintenanceWindowDeepLinkIds) => {
     services: {
       chrome: { docTitle, setBreadcrumbs },
       application: { navigateToUrl },
+      serverless,
     },
   } = useKibana();
+
   const setTitle = docTitle.change;
   const { getAppUrl } = useNavigation(MANAGEMENT_APP_ID);
 
   useEffect(() => {
     const breadcrumbs = [
-      {
```

Comment:
This removes the root breadcrumb from the non serverless version too. Given that the rest of pages show the Stack Management root crumb I'm pretty sure that this is not the desired behaviour

<img width="726" alt="Screenshot 22" src="https://github.com/user-attachments/assets/3a589cbf-2386-465a-ba3b-8ea8aca40d40">

Reply:
I added a condition that the 'Stack Management' breadcrumb to be rendered on the non serverless version only. Thanks!